.github/workflows/build.yml

```
@@ -166,11 +166,17 @@ jobs:
               \`\`\`
               `;
             }, '' );
+
+            let jetpackMuWPcomLine = '';
+            if ( data.projects.includes( 'jetpack-mu-wpcom-plugin' ) ) {
+              jetpackMuWPcomLine = "- For \`jetpack-mu-wpcom\` changes, also add \`define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );\` to your \`wp-config.php\` file.";
```

I think it'll look better if the line is indented.
```suggestion
              jetpackMuWPcomLine = "  - For \`jetpack-mu-wpcom\` changes, also add \`define( 'JETPACK_MU_WPCOM_LOAD_VIA_BETA_PLUGIN', true );\` to your \`wp-config.php\` file.";
```